reo/nested_inputs.py

```diff
@@ -289,7 +296,7 @@ def list_of_dict(input):
         "annual_kwh": {
           "type": "float",
           "min": 1.0,
-          "max": max_big_number,
+          "max": 1.0E10,
```

Comment:
Why did the `annual_kwh` max input need to be increased? (Maybe we should consider scaling our problems to megawatts?).

Reply:
I think we had a large DoD site that exceeded the ~11.4 MW average load, so I increased the max number up by 2 orders of magnitude to be safe.